mindmeld/components/question_answerer.py

```diff
@@ -256,7 +283,26 @@ def load_kb(
                 connection to the Elasticsearch host.
             clean (bool): Set to true if you want to delete an existing index
                 and reindex it
+            app_path (str): The path to the directory containing the app's data
         """
+        embedder_model = None
```

Comment:
maybe we should refactor this into a `load_emebedder_model` function

Reply:
will take a look

Reply:
fixed